### DISCUSSION TITLE
recommended settings for phospho data

### DISCUSSION THREAD

**Discussion by DanielVogele** [vdemichev/DiaNN#519] [Q&A]
Hello,

I've analysed a dataset of 200 phosphorylated and unphosphorylated peptides. As the sequences and phosphosites are known, I can determine how many are correctly annotated by DIA-NN. From potentially 200 phosphopeptides, only 53 were identified with the correct phosphosite after applying a cut-off of "PTM site confidence" ≥ 1 and "PTM Q-Value" ≤ 0.05. And there were still 5 peptides identified with a incorrect phosphosite. This strict cut-off was chosen because a large number of incorrectly identified phosphopeptides were seen without it.

Compared to DDA data, which was aquired in parallel and analysed with Fragpipe, these are quite underwhelming results.

Are there any recommended setting to analyse phospho data and potentially improve these results?
This data was analysed by first generating a spectral library from a FASTA file (only containing the used peptides), which was then used to analyse the actual .raw.dia data with MbR.

best,

Daniel

**Comment by vdemichev** [comment]
Hi Daniel,

What is the background in which the peptides were spiked in? How does the DIA-NN log look like, for all steps including predicted library generation? The results are guaranteed to be suboptimal if only 200 peptides are included in the library, without including background peptides. How do the FragPipe results look like? What LC-MS setup was used?

Best,
Vadim

**Reply by DanielVogele** [reply]
Hi Vadim,

sorry for the late response. In this experiment there was no background added to the peptides, only peptides of interest were processed by phospho-enrichment and measured subsequently. In the meantime I also analysed a similar experiment with _E .coli_ background with the same selection of peptides. The results are similar to the initial results. 
Find attached the summarised results: 
 - Exp1_Fragpipe_vs_DIANN (tables are from fragpipe DDA and barchart from DIA-NN analysis)
 - Exp2_with_Ecoli_background (DIA-NN analysis of selected peptides with _E. coli_ background, filtered for spiked peptides)
 
![Exp1_Fragpipe_vs_DIANN](https://user-images.githubusercontent.com/105349486/197740391-8a3f6fca-68f8-44ce-b4dd-e8db1c7a2adf.PNG)
![Exp2_with_Ecoli_background](https://user-images.githubusercontent.com/105349486/197740396-3d2f4e6a-b78c-4aae-a3ae-7148b5112994.PNG)

 
Also find attached the log files of both:
[Exp1_library.log.txt](https://github.com/vdemichev/DiaNN/files/9859092/Exp1_library.log.txt)
[Exp1_analysis.log.txt](https://github.com/vdemichev/DiaNN/files/9859093/Exp1_analysis.log.txt)
[Exp2_library.log.txt](https://github.com/vdemichev/DiaNN/files/9859094/Exp2_library.log.txt)
[Exp2_analysis.log.txt](https://github.com/vdemichev/DiaNN/files/9859095/Exp2_analysis.log.txt)
 experiments.

In terms of LC-MS setup: We are using a 200 cm µPAC analytical column including a trapping column beforehand. Measuremens were done on a QE plus with our well established 24 m/z staggared windows DIA methode. We did not adjust any parameters for these experiments.

Best,
Daniel

**Comment by vdemichev** [comment]
Hi Daniel,

Sorry, what are enriched 1/2, unprocessed 1/2? The results in tables are DDA? Results on plots are DIA? 

Some comments:
- MBR was not used for DIA-NN
- With MBR you don't want to filter or maybe filter using site confidence > 0.75 or maybe 0.9. No other filters.
- Max var mods was set to 1 in DIA-NN. Do all peptides only have a single phosphate?

Best,
Vadim
